library/swift/src/filters/Filter.swift

```diff
@@ -105,5 +105,55 @@ extension EnvoyHTTPFilter {
         }
       }
     }
+
+    if let asyncRequestFilter = filter as? AsyncRequestFilter {
+      self.setRequestFilterCallbacks = { envoyCallbacks in
+        asyncRequestFilter.setRequestFilterCallbacks(
```

Comment:
`asyncRequestFilter` is being retained here - I assume that's intentional, but want to flag

Reply:
Yep, good callout, but should be fine since both the internal filter and downcasted reference are both tied to the lifecycle of the parent Filter.